tests/texlive/latexdiff.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test the latexdiff command with a simple example
+# Maintainer: Katerina Lorenzova <klorenzova@suse.cz>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console('root-console');
+    zypper_call('in texlive-latexdiff-bin', timeout => 1800);
```

Comment:
The x11 tests commonly use `ensure_installed`, maybe that is a better match here and will also work without a need to switch consoles back and forth

Reply:
I tried `ensured_installed` when I was writing this test, but it didn't work. Installing all latex packages takes so long that computer lock itself. The needle at the end of `ensured_installed` is checked against login screen.
I decided to use textmode installation to prevent screen from locking.

Reply:
I see. Ok, keep this part then :)